template/lambda-sqs-worker-cdk/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14-alpine3.11 as builder
+ARG NPM_TOKEN
+WORKDIR /app
+COPY ./ /app
```

Comment:
This is actually problematic w.r.t `docker-ecr-cache` because your source code is not decoupled from cached dependencies. You could end up deploying stale code from the ECR cache if `package.json` and `yarn.lock` haven't change. You'll have better luck copying this directly from the `lambda-sqs-worker` template which correctly caches.

Reply:
Thanks! Fixed it up as suggested.